.circleci/config.yml

```diff
@@ -49,83 +34,20 @@ commands:
     steps:
       - run:
           name: Install Node dependencies
-          command: npm install
-
-  test_lib_transpilation:
-    steps:
-      - run:
-          name: Test lib transpilation
-          command: npm run lib
-
-  test_lib_import:
-    steps:
-      - run:
-          name: Test JS library imports
-          command: |
-            npm run lib-test:setup
-            cd tools/test-lib/react-app
-            npm run test:ci
-
-  run_eslint:
-    steps:
-      - run:
-          name: Run Eslint
-          command: npm run lint
-
-  run_javascript_tests:
-    steps:
-      - run:
-          name: Run JavaScript tests
-          command: npm run build:css && npm run test:ci
-
-  run_python_format_and_lint:
-    steps:
-      - run:
-          name: Run Python formatters and linters
-          command: make format-check lint-check
-
-  run_secret_scan:
-    steps:
-      - run:
-          name: Run secret scan
-          command: make secret-scan
+          command: npm install            
 
-  run_security_scan:
-    steps:
-      - run:
-          name: Run security scan
-          command: make security-scan
-
-  run_python_tests:
-    steps:
-      - run:
-          name: Run Python tests
-          command: make pytest
-
-  run_e2e_tests:
-    steps:
-      - run:
-          name: Run all end to end tests
-          command: make e2e-tests
-
-  cleanup:
-    steps:
-      - run:
-          name: Cleanup
-          command: make clean
-
-  build_npm_package:
+  build_package:
```

Comment:
Let's try to give this and `package_app` names which make it clearer what the difference is. Maybe `npm_build` and `python_package`?